### ISSUE TITLE
devmap makes the game explode

### ISSUE THREAD

**Issue by kharnov** [Unvanquished/Unvanquished#824]
I can't consistently reproduce this, but it's too weird to ignore. All I did was just load the game and type /devmap plat23 on the main menu.
```
>Program received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fffc0da1700 (LWP 4051)]
0x00007fffbec60260 in lua_close () from /usr/lib/x86_64-linux-gnu/liblua5.2.so.0
(gdb) bt
#0  0x00007fffbec60260 in lua_close () from /usr/lib/x86_64-linux-gnu/liblua5.2.so.0
#1  0x00007fffb632fee7 in Rocket_Shutdown () at /home/kharnov/Unvanquished/src/cgame/rocket/rocket.cpp:410
#2  0x00007fffb62f2b3e in CG_Shutdown () at /home/kharnov/Unvanquished/src/cgame/cg_main.cpp:1765
#3  0x00007fffb62c8d35 in VM::<lambda()>::operator() (__closure=<optimized out>) at /home/kharnov/Unvanquished/src/cgame/cg_api.cpp:71
#4  Util::apply_impl<VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda()>, std::tuple<> > (func=<optimized out>, tuple=<optimized out>) at /home/kharnov/Unvanquished/daemon/src/common/Util.h:103
#5  Util::apply<VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda()>, std::tuple<> > (func=<optimized out>, tuple=<optimized out>) at /home/kharnov/Unvanquished/daemon/src/common/Util.h:108
#6  IPC::detail::HandleMsg<VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda()>, IPC::Message<IPC::Id<0u, 2u> >, IPC::Reply<> > (channel=..., func=<optimized out>, reader=...)
    at /home/kharnov/Unvanquished/daemon/src/common/IPC/Channel.h:217
#7  IPC::HandleMsg<IPC::SyncMessage<IPC::Message<IPC::Id<0u, 2u> > >, VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda()> > (channel=..., func=<optimized out>, 
    reader=<error reading variable: access outside bounds of object referenced via synthetic pointer>) at /home/kharnov/Unvanquished/daemon/src/common/IPC/Channel.h:241
#8  VM::VMHandleSyscall (id=id@entry=2, reader=...) at /home/kharnov/Unvanquished/src/cgame/cg_api.cpp:72
#9  0x00007fffb63939d5 in IPC::detail::SendMsg<void (&)(unsigned int, Util::Reader), IPC::Message<IPC::Id<(unsigned short)6, (unsigned short)0>>, IPC::Reply<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > > >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >&, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > >&>(IPC::Channel&, void (&)(unsigned int, Util::Reader), IPC::SyncMessage<IPC::Message<IPC::Id<(unsigned short)6, (unsigned short)0>>, IPC::Reply<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > > > >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >&, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > >&) (channel=..., messageHandler=
    @0x7fffb62c8460: {void (unsigned int, Util::Reader)} 0x7fffb62c8460 <VM::VMHandleSyscall(unsigned int, Util::Reader)>) at /home/kharnov/Unvanquished/daemon/src/common/IPC/Channel.h:174
#10 0x00007fffb638bda5 in IPC::SendMsg<IPC::SyncMessage<IPC::Message<IPC::Id<(unsigned short)6, (unsigned short)0>>, IPC::Reply<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > > > >, void (&)(unsigned int, Util::Reader), std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >&, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > >&>(IPC::Channel&, void (&)(unsigned int, Util::Reader), std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >&, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > >&) (
    messageHandler=<optimized out>, channel=...) at /home/kharnov/Unvanquished/daemon/src/common/IPC/Channel.h:234
#11 VM::SendMsg<IPC::SyncMessage<IPC::Message<IPC::Id<(unsigned short)6, (unsigned short)0>>, IPC::Reply<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > > > >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >&, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > >&>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::vector<FS::PakInfo, std::allocator<FS::PakInfo> >&, std::vector<FS::LoadedPakInfo, std::allocator<FS::LoadedPakInfo> >&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<unsigned int, long>, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::pair<unsigned int, long> > > >&) () at /home/kharnov/Unvanquished/daemon/src/shared/VMMain.h:48
#12 FS::Initialize () at /home/kharnov/Unvanquished/daemon/src/common/FileSystem.cpp:2043
---Type <return> to continue, or q <return> to quit---
#13 0x00007fffb62c8b40 in VM::<lambda(int)>::operator() (__closure=<optimized out>, milliseconds=30823) at /home/kharnov/Unvanquished/src/cgame/cg_api.cpp:56
#14 Util::apply_impl<VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda(int)>, std::tuple<int&&>, 0ul> (func=<optimized out>, tuple=<optimized out>) at /home/kharnov/Unvanquished/daemon/src/common/Util.h:103
#15 Util::apply<VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda(int)>, std::tuple<int&&> > (func=<optimized out>, tuple=<optimized out>) at /home/kharnov/Unvanquished/daemon/src/common/Util.h:108
#16 IPC::detail::HandleMsg<VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda(int)>, IPC::Message<IPC::Id<0u, 0u>, int>, IPC::Reply<> > (channel=..., func=<optimized out>, reader=...)
    at /home/kharnov/Unvanquished/daemon/src/common/IPC/Channel.h:217
#17 IPC::HandleMsg<IPC::SyncMessage<IPC::Message<IPC::Id<0u, 0u>, int> >, VM::VMHandleSyscall(uint32_t, Util::Reader)::<lambda(int)> > (channel=..., func=<optimized out>, 
    reader=<error reading variable: access outside bounds of object referenced via synthetic pointer>) at /home/kharnov/Unvanquished/daemon/src/common/IPC/Channel.h:241
#18 VM::VMHandleSyscall (id=<optimized out>, reader=...) at /home/kharnov/Unvanquished/src/cgame/cg_api.cpp:59
#19 0x00007fffb637f3af in CommonInit (rootSocket=<optimized out>) at /home/kharnov/Unvanquished/daemon/src/shared/VMMain.cpp:66
#20 0x00007fffb637f64b in vmMain (rootSocket=<optimized out>) at /home/kharnov/Unvanquished/daemon/src/shared/VMMain.cpp:104
#21 0x00000000005981af in VM::<lambda()>::operator() (__closure=0x71007d8) at /home/kharnov/Unvanquished/daemon/src/engine/framework/VirtualMachine.cpp:352
#22 std::_Bind_simple<VM::CreateInProcessNativeVM(std::pair<IPC::Socket, IPC::Socket>, Str::StringRef, VM::VMBase::InProcessInfo&)::<lambda()>()>::_M_invoke<> (this=0x71007d8)
    at /usr/include/c++/5/functional:1531
#23 std::_Bind_simple<VM::CreateInProcessNativeVM(std::pair<IPC::Socket, IPC::Socket>, Str::StringRef, VM::VMBase::InProcessInfo&)::<lambda()>()>::operator() (this=0x71007d8)
    at /usr/include/c++/5/functional:1520
#24 std::thread::_Impl<std::_Bind_simple<VM::CreateInProcessNativeVM(std::pair<IPC::Socket, IPC::Socket>, Str::StringRef, VM::VMBase::InProcessInfo&)::<lambda()>()> >::_M_run(void) (this=0x71007c0)
    at /usr/include/c++/5/thread:115
#25 0x00007ffff4656030 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#26 0x00007ffff3e676aa in start_thread (arg=0x7fffc0da1700) at pthread_create.c:333
#27 0x00007ffff3b9ceed in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:109
```

**Comment by Viech** [comment]
@kharnov, reason for closing?

**Comment by Kangz** [comment]
I think it was fixed by 79fc559f268f277e98636e058ca558a239cb9e63
